engine/internal/runtime/client.go

```diff
@@ -275,3 +276,13 @@ func (c *commonClient) deployRuntime(
 	}
 	return nil
 }
+
+func statefulSetName(runtime, modelID string) string {
```

Comment:
I feel `resourceName` would be nicer since this name is also used for SVC and PVC.

Reply:
done!